test/cases/opentelemetry/conf/otel_directives.conf

```diff
@@ -0,0 +1,47 @@
+# "/datadog-tests" is a directory created by the docker build
+# of the nginx test image. It contains the module, the
+# nginx config, and "index.html".
+load_module /datadog-tests/ngx_http_datadog_module.so;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    opentelemetry_service_name "foo";
+    opentelemetry_otlp_traces_endpoint http://my_agent:8015;
+
+    server {
+        listen       80;
+
+        opentelemetry_operation_name "operation";
+        opentelemetry_trust_incoming_spans on;
+        opentelemetry_traces_sampler_ratio 1.0;
+        opentelemetry_attribute "my_otel_tag" "otelotel";
+
+        # make sure ignored commands does nothing
+        opentelemetry_propagate b3;
+        opentelemetry_span_processor "simple";
+        opentelemetry_bsp_max_queue_size 1;
+        opentelemetry_bsp_schedule_delay_millis 0;
+        opentelemetry_bsp_max_export_batch_size 128;
+        opentelemetry_traces_sampler always_on;
+        opentelemetry_capture_headers on;
+        opentelemetry_sensitive_header_names "x-datadog-trace-id";
+        opentelemetry_sensitive_header_values "value";
+        opentelemetry_ignore_paths "ignored.php";
```

Comment:
I'm going to be picky (sorry for that) but are these aliases tested as well ?

Reply:
They may be untestable though

Reply:
These directives are silently ignored, as long as it does prevent nginx to start, we are good